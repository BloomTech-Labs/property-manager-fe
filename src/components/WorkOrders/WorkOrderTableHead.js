import React from 'react';
import { TableCell, TableHead, TableRow } from '@material-ui/core';

function WorkOrderTableHead() {
  return (
    <>
      <TableHead>
        <TableRow>
          <TableCell align="left">Work&nbsp;Order</TableCell>
          <TableCell align="left">Description</TableCell>
          <TableCell align="left">Type</TableCell>
          <TableCell align="left">Start&nbsp;Date</TableCell>
          <TableCell align="left">Property</TableCell>
          <TableCell align="left">Created&nbsp;By</TableCell>
          <TableCell align="left">Status</TableCell>
          <TableCell>Actions</TableCell>
        </TableRow>
      </TableHead>
    </>
  );
}

export default WorkOrderTableHead;
