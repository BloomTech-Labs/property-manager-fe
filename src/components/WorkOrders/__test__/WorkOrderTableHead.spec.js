import React from 'react';
import * as rtl from '@testing-library/react';
import '@testing-library/jest-dom';
import WorkOrderTableHead from '../WorkOrderTableHead';

afterEach(rtl.cleanup);

test('Makes sure the word type exists', async () => {
  const wrapper = rtl.render(<WorkOrderTableHead />);
  expect(await wrapper.queryAllByText(/type/i));
});

test('snapshot', () => {
  const { container } = rtl.render(<WorkOrderTableHead />);
  expect(container.firstChild).toMatchSnapshot();
});
