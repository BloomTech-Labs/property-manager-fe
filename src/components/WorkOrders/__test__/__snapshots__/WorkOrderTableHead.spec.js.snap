// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
<thead
  class="MuiTableHead-root"
>
  <tr
    class="MuiTableRow-root MuiTableRow-head"
  >
    <th
      class="MuiTableCell-root MuiTableCell-head MuiTableCell-alignLeft"
      scope="col"
    >
      Work Order
    </th>
    <th
      class="MuiTableCell-root MuiTableCell-head MuiTableCell-alignLeft"
      scope="col"
    >
      Description
    </th>
    <th
      class="MuiTableCell-root MuiTableCell-head MuiTableCell-alignLeft"
      scope="col"
    >
      Type
    </th>
    <th
      class="MuiTableCell-root MuiTableCell-head MuiTableCell-alignLeft"
      scope="col"
    >
      Start Date
    </th>
    <th
      class="MuiTableCell-root MuiTableCell-head MuiTableCell-alignLeft"
      scope="col"
    >
      Property
    </th>
    <th
      class="MuiTableCell-root MuiTableCell-head MuiTableCell-alignLeft"
      scope="col"
    >
      Created By
    </th>
    <th
      class="MuiTableCell-root MuiTableCell-head MuiTableCell-alignLeft"
      scope="col"
    >
      Status
    </th>
    <th
      class="MuiTableCell-root MuiTableCell-head"
      scope="col"
    >
      Actions
    </th>
  </tr>
</thead>
`;
